app/dashboard/models.py

```diff
@@ -905,7 +905,7 @@ def value_in_usdt_now(self):
         if self.tokenName == 'USDT':
             return float(self.amount)
         if self.tokenName == 'DAI':
```

Comment:
Can you move the `if self.tokenName == 'DAI'`to `if self.tokenName == 'USDT'` via `if self.tokenName in ['USDT', 'DAI']:` in both cases?